plugin/storage/cassandra/spanstore/operation_names.go

```diff
@@ -25,23 +26,51 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/cache"
 	"github.com/jaegertracing/jaeger/pkg/cassandra"
 	casMetrics "github.com/jaegertracing/jaeger/pkg/cassandra/metrics"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 const (
-	insertOperationName = `INSERT INTO operation_names(service_name, operation_name) VALUES (?, ?)`
-	queryOperationNames = `SELECT operation_name FROM operation_names WHERE service_name = ?`
+	// LatestVersion latest version of operation_names table schema, increase the version if your table schema changes require code change
+	LatestVersion = 1
```

Comment:
The semantics of this are not clear to me, w.r.t. the schemaMeta below. E.g. the new table is called operation_names_v2, but its index in the table is `1`.

What if you do LatestVersion as a string, e.g. `v2`, and change the below table to a `map[strong]schemaMeta`